R/assertions.R

```diff
@@ -270,12 +270,17 @@ assert_character_vector <- function(arg, values = NULL, named = FALSE, optional
 #' Checks if an argument is a logical scalar
 #'
 #' @param arg A function argument to be checked
-#'
-#' @param optional Is the checked argument optional?
-#'
+#' @param optional Is the checked argument optional?\cr
 #' If set to `FALSE` and `arg` is `NULL` then an error is thrown. Otherwise,
 #' `NULL` is considered as valid value.
-#'
+#' @param arg_name (`string`)\cr
+#'   string indicating the label/symbol of the object being checked.
+#'   Default is `rlang::caller_arg(x)`
+#' @param message (`character`)\cr
+#'   string passed to `cli::cli_abort(message)`. When `NULL`, default messaging
```

Comment:
"When `NULL`, default ..." could be removed if we set the default value in the formals.

Reply:
I think we should just keep the default value as NULL, because the message is too long for the package manual and results in NOTES during R CMD Check.